### ISSUE TITLE
BUG in literalList widget

### ISSUE THREAD

**Issue by SteinerPascal** [SteinerPascal/sparnatural-port#108]
Currently in the literal list widget what we produce when selecting a value is this.
```
SELECT DISTINCT ?Person_1 WHERE {
  ?Person_1 rdf:type <http://dbpedia.org/ontology/Person>.
  ?Country_2 rdf:type <http://dbpedia.org/ontology/Country>.
  ?Person_1 <http://fr.dbpedia.org/property/nationalité> ?Country_2.
  VALUES ?Country_2 {
    <-  Allemand>
  }
}
```

But the <-Allemand> is not a valid value.
What should be the correct value?

**Comment by tfrancart** [comment]
Fixed in #112

**Comment by SteinerPascal** [comment]
merged in [e08d0c2](https://github.com/SteinerPascal/sparnatural-port/commit/e08d0c2f0fcf141d5d1d2cd12358cffa72cabbe9)
